sidebarsAstro.js

```diff
@@ -124,6 +124,46 @@ module.exports = {
         },       
       ],
     },
+    {
+      type: "category",
+      label: "Automation",
```

Comment:
I am thinking instead of automation, we should name it as "Programmatic access"
```suggestion
      label: "Programmatic access",
```